pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -463,6 +463,7 @@ func manageAPIServices_v311_00_to_latest(client apiregistrationv1client.APIServi
 				VersionPriority:      15,
 			},
 		}
+		apiregistrationv1.SetDefaults_ServiceReference(obj.Spec.Service)
```

Comment:
oh, you are lucky here. Defaults are considered private to the API server. But we make them accessible by accident.